psi4/driver/procrouting/proc.py

```diff
@@ -1954,16 +1958,70 @@ def run_occ_gradient(name, **kwargs):
 
 
 def run_scf(name, **kwargs):
-    """Function encoding sequence of PSI module calls for
+    """
```

Comment:
is yapf doing this? I rather like 
https://github.com/numpy/numpy/blob/master/doc/HOWTO_DOCUMENT.rst.txt#docstring-standard